scripts/check_register.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+import csv
+from collections import defaultdict
+import operator
+import argparse
+import sys
+import os
+import get_ps_list
+
+def parse_args(args_):
+    parser = argparse.ArgumentParser(
+        description='Check any mismatch on registers.'
+                    ' Written by Yazhou He.'
+    )
+    parser.add_argument(
+        '-preservation_storage_csv',
+        help='the correct mapping csv'
```

Comment:
A more meaningful description here would be better

Reply:
updated - (Optional) The correct mapping csv generated from get_ps_list.py. Script will get one if not.